src/yaml/sg/simtropolis-train-station.yaml

```diff
@@ -0,0 +1,46 @@
+group: sg
+name: simtropolis-train-station
+version: "1.0"
+subfolder: 700-transit
+info:
+  summary: SimTropolis Train Station
+  description: |-
+    A modern passenger station designed for use in a downtown setting. The upper promenade features boutique shops and restaurants, and there are offices in the front glassed in building. There are three tracks off the rear, 3 tiles back from the road, so it can be used in the middle of a commercial district.
+
+     **Lot Details and Information**
+
+     - Building: Transportation
+     - Lot Size: 7 x 6 tiles
+     - Lot Wealth: Medium Wealth
+     - Jobs: Civic $: 150, $$: 70, $$$: 30
+  author: SimGoober
+  website: https://community.simtropolis.com/files/file/14728-simtropolis-train-station/
+  images:
+    - https://www.simtropolis.com/objects/screens/0023/d5617fc0dc0a2a67c3cf3fbb7bfbbc94-SG_SimTropolisStation01.jpg
+    - https://www.simtropolis.com/objects/screens/0023/d5617fc0dc0a2a67c3cf3fbb7bfbbc94-SG_SimTropolisStation02.jpg
+dependencies:
+  - bsc:essentials
+  - bsc:textures-vol01
+  - bsc:textures-vol02
+  - bsc:mega-props-misc-vol01
+  - bsc:mega-props-sg-vol01
```

Comment:
bsc:textures-vol02 and bsc:mega-props-misc-vol01 are not listed on the STEX entry.

Also try to avoid unnecessary indentation in the description in case it's not on purpose.

Reply:
They are indeed not, but my dependency tracking script found missing dependencies. I verified it manually and indeed, some trees and textures did not show up when I only used the listed dependencies. That's why I added them here.

Reply:
Ok, thanks. I've sent a STEX report about it.